admin/resources/js/components/UserTable.tsx

```diff
@@ -31,17 +35,34 @@ const UserTable: React.FC<AllUsersQuery> = ({ users }) => {
         Header: "Preferred Language",
         accessor: "preferredLang",
       },
+      {
+        Header: "",
+        id: "edit",
+        accessor: ({ id }) => (
+          <Link href={`${editUrlRoot}/${id}/edit`} title="">
+            Edit
+          </Link>
+        ),
+      },
     ],
-    [],
+    [editUrlRoot],
   );
 
   const data = useMemo(() => users.filter(notEmpty), [users]);
 
   return (
     <>
-      <Table data={data} columns={columns} />
+      <Table<User> data={data} columns={columns} />
     </>
   );
 };
 
-export default UserTable;
+export const ApiUserTable: React.FunctionComponent = () => {
```

Comment:
Was the convention changed from appending Api to prepending Api to the original component name? If not, this should be UserTableApi and any references to it changed accordingly.